doc/developer/platform/ux.md

```diff
@@ -1,20 +1,945 @@
+any downtime. A sufficiently intelligent materialize-dbt adapter could create
+the new views in a new cluster, wait for them to come up to date with the old
+views, then atomically swap the clusters.
+
+Materialize Platform will also offer one-click integration with dbt cloud.
+
+### Other PostgreSQL-compatible drivers and tools
+
+Like Materialize Cloud today, Materialize Platform will support as many
+PostgreSQL-compatible tools as possible. Each tool will need to be evaluated on
+a case-by-case basis, as tools often require small feature additions or bug
+fixes to Materialize.
+
+It may not be possible to support all features of a tools. In rare cases,
+building a seamless integration with a tool may involve building a custom,
+Materialize-native plugin for that tool.
+
+Prioritizing which tools to support is outside the scope of this document.
+
+### Terraform
+
+dbt is essentially Terraform for databases. By supporting dbt, we eliminate much
+of the need for Terraform.
+
+However, while dbt has excellent support for managing views and indexes, and good
+support for managing sources and sinks, it is not the ideal tool for managing
+clusters, roles, databases, schemas, or secrets. So we might want to build a
+Terraform provider to manage these resources types.
+
+## APIs
+
+### Admin API
+
+The admin API is a REST API hosted at <https://admin.cloud.materialize.com>.
+It is provided by [Frontegg], our user management provider, and handles
+authentication and configuration of account security policies.
+
+The [Frontegg API reference][frontegg-api] specifies this API.
+
+The primary operation that this API provides is the exchange of user credentials
+for a short-lived authentication token. User credentials may be a username and
+password or an API key.
+
+We don't plan to publicize or document this API, but we don't plan to hide it,
+either.
+
+### Global API
+
+The global API is a REST API hosted at <https://cloud.materialize.com/api>. It
+handles operations that apply globally to an account rather than to a single
+region within an account.
+
+To authenticate with the global API, users supply an authentication token
+obtained from the admin API.
+
+We will make the following changes to the [existing API specification][global-api-spec]:
+
+  * Deprecate all operations on the `deployments` resource.
+  * Deprecate all operations on the `mz-versions` resource.
+  * Add a new API to enable a cloud-provider region for an account:
+
+    `PUT /api/region/{cloud-provider}/{region}`
+
+In the future, we will add billing operations to this API.
+
+As with the admin API, we don't plan to publicize or document the global API,
+but we don't plan to hide it, either.
+
+### Region API
+
+The user-facing API for Materialize Platform is the region API. This is the
+only API that we will actively advertise to users. The intent is that users
+think of the region API as **the** Materialize Platform API.
```

Comment:
to clarify: this one endpoint covers ALL interactions with Materialize, both DDLs like creating views and sources, as well as all queries/app traffic?

Reply:
That's exactly right. (With the _one_ little exception that you have to send `EnableRegion` API calls to the global endpoint rather than a regional SQL endpoint.)